src/account/monster.js

```diff
@@ -1,8 +1,8 @@
 const SqlString = require('sqlstring');
-
+import { fromRawLsk } from '../helpers/lisk';
```

Comment:
If you import and export, this file should be a typescript one instead of js?

Reply:
Because of compatibility? Okay I'll change it

Reply:
You can't use the keywork `import` and `export` in js files otherwise the production build will be broken :(